### PR TITLE
Fix the diagram and improve documentation

### PR DESCRIPTION
**Pull request by bszwarc**
**Description**

Changes proposed in this pull request:

- Fix the diagram according to remarks in the task
- Improve documentation (fix typos/capitalization)
- ...

**Related issue(s)**
See also #2345

### CODE REVIEW

File: docs/event-bus/docs/03-03-service-programming-model.md [hunk 1]
```diff
@@ -3,31 +3,31 @@ title: Service Programming Model
...
 
-The Event Delivery workflow is as follows:
+The Event delivery workflow is as follows:
 
-1. The Event is published to the Kyma Event Bus from an external system instance in a bound Application.
-2. The Event Bus checks for the Event subscription and activation. It creates an `HTTP POST` request using Event Payload and Metadata.
-3. The Service receives the `HTTP POST` request. The Event Metadata is represented in the `HTTP Request Headers` request and the Event Payload is represented in the `HTTP Request Body`.
+1. An external system uses the bound Application to publish an Event to the Kyma Event Bus.
+2. The Event Bus checks for the Event subscription and activation. It creates an `HTTP POST` request using Event payload and metadata.
+3. The service receives the `HTTP POST` request. `HTTP Request Headers` request represents Event metadata and `HTTP Request Body`- the Event payload.
```
Comment (kazydek) [inline docs/event-bus/docs/03-03-service-programming-model.md]:
```suggestion
3. The service receives the `HTTP POST` request. `HTTP Request Headers` request represents Event metadata, and `HTTP Request Body` represents the Event payload.
```

File: docs/event-bus/docs/03-03-service-programming-model.md [hunk 2]
```diff
@@ -3,31 +3,31 @@ title: Service Programming Model
 type: Details
 ---
 
-You can configure a microservice deployed in Kyma to receive Events from the Event Bus by creating a Subscription custom resource.
+You can configure a microservice deployed in Kyma to receive Events from the Event Bus by creating a ;[Subscription](/components/event-bus/#custom-resource-subscription) custom resource.
```
[the remark this answers is not in the document]
Comment (bszwarc) [reply docs/event-bus/docs/03-03-service-programming-model.md]: Done

File: docs/event-bus/docs/03-03-service-programming-model.md (hunk 1 above)
Comment (bszwarc) [reply docs/event-bus/docs/03-03-service-programming-model.md]: Done

File: docs/event-bus/docs/03-03-service-programming-model.md [hunk 3]
```diff
@@ -3,31 +3,31 @@ title: Service Programming Model
...
+![eventdelivery](./assets/event-delivery.svg)
 
-The Event Delivery workflow is as follows:
+The Event delivery workflow is as follows:
 
-1. The Event is published to the Kyma Event Bus from an external system instance in a bound Application.
-2. The Event Bus checks for the Event subscription and activation. It creates an `HTTP POST` request using Event Payload and Metadata.
-3. The Service receives the `HTTP POST` request. The Event Metadata is represented in the `HTTP Request Headers` request and the Event Payload is represented in the `HTTP Request Body`.
+1. An external system uses the bound Application to publish an Event to the Kyma Event Bus.
+2. The Event Bus checks for the Event subscription and activation. It creates an `HTTP POST` request using Event payload and metadata.
```
[the remark this answers is not in the document]
Comment (bszwarc) [reply docs/event-bus/docs/03-03-service-programming-model.md]: changed the formatting

**Pull request merged**